packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -5,21 +5,21 @@
 %global plugin_name scc_manager
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.1
+Version: 1.3.1
 Release: 1%{?foremandist}%{?dist}
 Summary: Suse Customer Center plugin for Foreman
 Group: Applications/Systems
 License: GPL-3.0
 URL: https://www.orcharhino.com/
 Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
-Requires: foreman >= 1.13.0
+Requires: foreman >= 1.16.0
```

Comment:
Shouldn't this be 1.17 now since it's rails 5?

Reply:
Yeah, agreed. Bumped.

Reply:
also upstream: https://github.com/ATIX-AG/foreman_scc_manager/pull/13